cross/libicu/patches/0001-Fixed-added-missing-rule-for-generating-OUTDIR.patch

```diff
@@ -0,0 +1,15 @@
+--- data/Makefile.in.old
++++ data/Makefile.in
+@@ -150,6 +150,10 @@ ifneq ($(ICUDATA_ARCHIVE),)
+ ICUDATA_SOURCE_ARCHIVE = $(OUTDIR)/$(ICUDATA_PLATFORM_NAME).dat
+ $(ICUDATA_SOURCE_ARCHIVE): $(ICUDATA_ARCHIVE) $(OUTDIR)
+ 	$(INVOKE) $(TOOLBINDIR)/icupkg -t$(ICUDATA_CHAR) $(ICUDATA_ARCHIVE) $(ICUDATA_SOURCE_ARCHIVE)
++	
++$(OUTDIR):
++	mkdir -p $(OUTDIR)
++
+ endif
+ else
+ ifneq ($(ENABLE_STATIC),YES)
+
```

Comment:
This has been proposed in PR on ICU project directly and is required for cross compilation purposes https://github.com/unicode-org/icu/pull/887.

Reply:
ICU dependency has been identified as no longer useful (since switching to cross/lxml usage, basically).